services/gas-api/api-reference/basefeehistory.md

```diff
@@ -32,20 +32,28 @@ Array of historical base fees.
 ### Request
```

Comment:
For each of the API requests, can you make the same changes as in the Infura PR? https://github.com/INFURA/docs/pull/234/files#diff-34a0f10aa9c71f132a388e20c0bd00d96f2c623954f89dcb183c9c386c1def49R32

(i.e., don't change the examples, just add a note pointing to the appropriate section)